src/test/ui/associated-const/lints-used-unused.used.stderr

```diff
@@ -0,0 +1,30 @@
+error: this arithmetic operation will overflow
+  --> $DIR/lints-used-unused.rs:16:20
+   |
+LL |     const N: i32 = 1 << 42;
+   |                    ^^^^^^^ attempt to shift left with overflow
+   |
+note: the lint level is defined here
+  --> $DIR/lints-used-unused.rs:7:9
+   |
+LL | #![deny(arithmetic_overflow)]
+   |         ^^^^^^^^^^^^^^^^^^^
+
+error: any use of this value will cause an error
+  --> $DIR/lints-used-unused.rs:16:20
+   |
+LL |     const N: i32 = 1 << 42;
+   |     ---------------^^^^^^^-
+   |                    |
+   |                    attempt to shift left with overflow
+   |
+   = note: `#[deny(const_err)]` on by default
```

Comment:
Does it make sense to try to deduplicate these errors? From a front-facing perspective, they're two different lints (`arithmetic_overflow` and `const_err`), and if someone wanted to silence these errors, they would have to disable both lints. What's the go for duplicates like this?